Anagram.java - Roobini

```diff
@@ -0,0 +1,55 @@
+import java.util.HashMap;
+import java.util.Map;
+
+
+public class Anagram {
+
+	public static boolean isAnagram(String input , String input1){
+		if(input == null || input1 == null){
+			return false;
+		}
+		if(input.length() == 0 && input1.length() == 0){
+			return true;
+		}
+		if(input.length() != input1.length()){
+			return false;
+		}
+		int count = 1;
+		Map<Character , Integer> myMap = new HashMap<Character , Integer>();
+		for(int i = 0; i < input.length() ; i++){
+			if(myMap.containsKey(input.charAt(i))){
+			myMap.put(input.charAt(i) , count+1);
+		}
+			else{
+				count = 1;
+				myMap.put(input.charAt(i) , count);
+			}
+		}
+		
+		for(int i = 0; i < input1.length() ; i++){
+			if(myMap.containsKey(input1.charAt(i))){
+			myMap.put(input1.charAt(i) , count-1);
+		}
+			else{
```

Comment:
This Else block is to reset the count to 1 when a new character comes which is not in the map.